lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -189,13 +183,12 @@ function dosomething_signup_sms_game_form($form, &$form_state, $node, $num_frien
     ),
   );
   for ($i = 0; $i < $num_friends; $i++) {
-    // Default to second fieldset.
-    $group = 'group2';
-    if ($i == 0) {
-      // Unless its the first beta.
-      $group = 'group1';
-    }
-    $form[$group]['beta_mobile_' .$i] = array(
+    $form['group1']['beta_first_name_' .$i] = array(
+      '#type' => 'hidden',
+      '#title' => 'Your Friend\'s Name',
```

Comment:
This field isn't actually doing anything besides telling Drupal to whitelist sending `'beta_first_name` fields along to Mobile Commons. The form fields used for the A/B test are hard-coded in the `<div id="ab-test-sms-names" />` markup in the template.